### PR TITLE
use Lando for solr

### PR DESCRIPTION
**Pull request by tpendragon**
Just playing with Lando as an option for dev dependencies.

### CODE REVIEW

**Comment by jeremyf** [comment]
Seems legible, though as someone that continues to develop bare metal, its a bit beyond my common usage. Though this next computer I'm getting will probably push me towards standardized development environments.

File: README.md [hunk 1]
```diff
@@ -101,6 +103,23 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
      - If you run into problems with `solr_wrapper`, you can also start Solr with `rake figgy:development`
    - Access Figgy at http://localhost:3000/
 
+### Lando
+
+1. Uninstall Docker
+2. Install Lando DMG from [[https://github.com/lando/lando/releases]]
```
Comment (hackmastera) [inline README.md:109]: What version of lando?

Comment (eliotjordan) [reply README.md:109]: I used the latest `v3.0.0-rrc.4` and it works well.

**Comment by tpendragon** [comment]
This used to have redis and I removed it. Maybe a follow-up?

File: .lando.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: figgy
+services:
+  figgy_test_solr:
+    type: solr:7
```
Comment (christinach) [inline .lando.yml:4]: is it still solr 7 for figgy?

Comment (hackmastera) [reply .lando.yml:4]: yep

File: README.md [hunk 3]
```diff
@@ -101,6 +103,23 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
...
+3. `lando start`
+
+1. For test:
+   - `RAILS_ENV=test rake db:setup`
+   - In a separate terminal: `bundle exec rspec`
+   - Run jest tests: `yarn test`
+2. For development:
+   - ``export SECRET_KEY_BASE=`rake secret` ``
+   - `rake db:setup`
+   - In a separate terminal: `foreman start` (you can close the one launching solr)
```
Comment (hackmastera) [inline README.md:119]: Procfile still includes solr wrapper. This will conflict with the lando port so it should probably be removed.

**Comment by hackmastera** [comment]
I left a couple of comments that can be addressed by documentation. Otherwise, the lando setup is nice but docker use is definitely taxing my system. I might need to work with it for a bit to gauge impact.

I don’t think I’d want to move other services for now (e.g. redis)

**Comment by christinach** [comment]
@tpendragon lando ran smoothly. My only concern is that docker consumes a high amount of CPU but maybe this is something I can improve in my local docker installation.

File: README.md (hunk 3 above)
Comment (tpendragon) [reply README.md:119]: It doesn't conflict with the lando port, but it isn't necessary so I'll remove it.

Comment (hackmastera) [reply README.md:119]: Hm. Not sure why foreman failed for me then. I don't generally use it so there may have been a pre-existing or unrelated issue. anyway 👍

**Pull request merged**